astacus/coordinator/plugins/cassandra/restore_steps.py

```diff
@@ -203,8 +203,14 @@ class WaitCassandraUpStep(Step[None]):
 
     async def run_step(self, cluster: Cluster, context: StepsContext) -> None:
         last_err = None
+
+        replaced_nodes = context.get_result(StopReplacedNodesStep)
```

Comment:
there's no StopReplacedNodes step on full restore, this would just fail with an exception in that case. Maybe instead of excluding the stopped nodes include only the new nodes? in full restore that'd be the whole cluster, in partial restore - only the nodes we restored.

Reply:
in full cluster restore there wont be any stopped nodes, so pinging all cluster.nodes should be okay (?)
so took an alternate approach of looking at the Stop step or just follow the defaults